Example/src/shared/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {Button as RNButton} from 'react-native';
+import {Spacer} from '.';
```

Comment:
Why this import looks like that?

Reply:
I've exported every file from `shared/` directory in `Example/src/shared/index.ts` file to have shorter imports in examples. Here, it literally means "import Spacer from here".